### PR TITLE
generic cmake

### PR DESCRIPTION
**Pull request by dzhwinter**

### CODE REVIEW

File: cmake/generic.cmake [hunk 1]
```diff
@@ -110,6 +110,20 @@ function(find_fluid_modules TARGET_NAME)
   endif()
 endfunction(find_fluid_modules)
 
+# find all third_party modules is used for paddle static library
+# for reduce the dependency when building the inference libs.
+set_property(GLOBAL PROPERTY FLUID_THIRD_PARTY)
+function(find_fluid_third_partys TARGET_NAME)
```
Comment (typhoonzero) [inline cmake/generic.cmake]: did not see where this function used. or is it in another PR?

`find_fluid_third_partys` => better name: `find_fluid_thirdparties`

Comment (dzhwinter) [reply cmake/generic.cmake]: follow comments.

Comment (dzhwinter) [reply cmake/generic.cmake]: It's used in inference_lib.cmake, which mess with a lot of windows commits, it was not included in this PR.

**Review by JiabinYang — APPROVED** [review]
LGTM

**Pull request merged**